projects/storefrontlib/src/shared/utils/validators/custom-form-validators.ts

```diff
@@ -102,6 +102,44 @@ export class CustomFormValidators {
 
     return validator;
   }
+
+  /**
+   * Checks if control's value is euqal or greater than 0
+   *
+   * NOTE: Use it as a control validator
+   *
+   * @static
+   * @param {AbstractControl} control
+   * @returns {(ValidationErrors | null)} Uses 'cxNegativeAmount' validator error
+   * @memberof CustomFormValidators
+   */
+  static mustBePositive(control: AbstractControl): ValidationErrors | null {
+    const amount = control.value as number;
+
+    return amount >= 0 ? null : { cxNegativeAmount: true };
+  }
+
+  /**
+   * Checks if control's value does not contain any special characters
+   *
+   * NOTE: Use it as a control validator
+   *
+   * @static
+   * @param {AbstractControl} control
+   * @returns {(ValidationErrors | null)} Uses 'cxContainsSpecialCharacters' validator error
+   * @memberof CustomFormValidators
+   */
+  static noSpecialCharacters(
```

Comment:
This validator checks only for slash. It should be renamed to `noSlash` or be more configurable.

Reply:
@psmul I assume there was no need for other characters.

Reply:
@tobi-or-not-tobi right, at the time of implementation it was only issued to be problem with slash. As @Platonn stated, we can improve it or rename it. Please let me know if you want me to take any actions against it.